### PR TITLE
Blacklist desktop profile settings; change edit profile link - fix #37

### LINKED ISSUE #37: Can't edit profile image with extension on
If you click on "Edit Profile" the form that you're given on the mobile version doesn't allow editing the profile image. It's also rather ugly.

I think it might be better to black list the settings URLs and allow users to edit their profile on the desktop version since it has a richer interface.

yucky

### PR DESCRIPTION
**Pull request by paulmolluzzo**
I added `waitFor` from `anatine` (I had to to get the timing right) and I changed the edit profile link so that you don't have to remember the URL.

This fixes #37

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -132,4 +160,7 @@ function init() {
 document.addEventListener('DOMContentLoaded', () => {
 	// TODO: figure out a better way to detect when React is done
 	setTimeout(init, 200);
```
Comment (sindresorhus) [inline extension/content.js]: We can remove this now and use the same selector we use in Anatine.

File: extension/content.js [hunk 2]
```diff
@@ -119,6 +143,10 @@ function applyMode(isDark) {
 	document.documentElement.classList.toggle('dark-mode', isDark);
 }
 
+function changeSettingsURL(linkElement) {
```
Comment (sindresorhus) [inline extension/content.js]: Just put this inline.

File: extension/content.js (hunk 1 above)
Comment (sindresorhus) [inline extension/content.js]: In a different commit of course.

File: extension/content.js (hunk 2 above)
Comment (paulmolluzzo) [inline extension/content.js]: If I do this as an arrow function I get an error for returning an assignment.

Comment (paulmolluzzo) [inline extension/content.js]: You mean do this instead, right?

```js
waitFor('.wRxmsUlJ a[href*="/settings/profile"]').then(el => el.href = el.href.replace('mobile.', ''););
```

Comment (sindresorhus) [inline extension/content.js]: Don't do an inline arrow function. Just `() => { code }`.

Comment (sindresorhus) [inline extension/content.js]:
```js
waitFor('.wRxmsUlJ a[href*="/settings/profile"]').then(el => {
  el.href = el.href.replace('mobile.', '');
});
```

File: extension/content.js (hunk 1 above)
Comment (paulmolluzzo) [inline extension/content.js]: You mean in this PR but a new commit or in a separate PR entirely?

Comment (sindresorhus) [inline extension/content.js]: Just commit it directly afterwards.

Comment (paulmolluzzo) [inline extension/content.js]: Pushed on `master`.

**Comment by jorgegonzalez** [comment]
> Refined Github let us disable features, any reason why Refined Twitter doesn't?

The work just hasn’t been done. I think we’re open to a PR implementing this if you’d like to help.

**Pull request merged**